### PR TITLE
Update Asgardia.Constitution.Proposal.txt

### PR DESCRIPTION
**Pull request by BloodyClean**
Grammar/wording updates for new changes

### CODE REVIEW

File: Asgardia.Constitution.Proposal.txt [hunk 1]
```diff
@@ -75,7 +75,7 @@ Article 4. Supreme Values of Asgardia
  l. fairness, and freedom;
  m. harmonious existence of the individual, society, and the Space Nation.
 § 5. Asgardia’s Supreme Values are equally important and equally implemented in state administration. In the event of a conflict between the Supreme Values, the absolute Supreme Value of Asgardia prevails.
-§ 6. Asgardia’s Supreme Values apply to relations with Earth nation states and citizens, and any other non-Asgardian territories in space, as well as any discovered alien intelligence.
+§ 6. Asgardia’s Supreme Values apply to relations with Earth nation states and citizens, and any other non-Asgardian territories and peoples in space.
```
Comment (BloodyClean) [inline Asgardia.Constitution.Proposal.txt:78]: Trying to simplify but retain same message

File: Asgardia.Constitution.Proposal.txt [hunk 2]
```diff
@@ -160,7 +160,7 @@ Article 12. Natural Resources
 Article 13. Financial Resources
 § 1. Asgardia’s sovereign financial resources are comprised of state and private finance. Arbitrary seizure of private financial resources is prohibited.
 § 2. Asgardia’s currency is the Solar. The symbols for Solar will be established by Asgardia’s National Bank. Solar shall not be represented with a decimal place or fractional value.
-§ 3. Asgardian currency is freely exchangeable into all of the main global currencies on the free market on Earth, and into any market will be established in space, planets, or other space bodies in the time.
+§ 3. Asgardian currency is freely exchangeable into all of the main global currencies on the free markets on Earth, and into any market will be established in space or celestial bodies in the future.
```
Comment (BloodyClean) [inline Asgardia.Constitution.Proposal.txt:163]: Trying to simplify but retain same message

File: Asgardia.Constitution.Proposal.txt [hunk 3]
```diff
@@ -359,15 +358,15 @@ Article 32. Asgardia’s Head of State
  d. Ministry of Administrative Affairs Chairperson
  e. Ministry of Justice Chairperson
 § 9. The Head of State:
- a. appoints the Supreme Justice of the Judicial Court, the Chairperson of the National Audit Office, the Chairperson of the Council of Supreme Values, Asgardian ambassadors extraordinary and plenipotentiary to Earth nation states, and any other space nation states;
+ a. appoints the Supreme Justice of the Judicial Court, the Chairperson of the National Audit Office, the Chairperson of the Council of Supreme Values, and Asgardian ambassadors extraordinary and plenipotentiary to Earth nation states, and any other space nation states;
```
Comment (BloodyClean) [inline Asgardia.Constitution.Proposal.txt:361]: The "Asgardian ambassadors..." portion of the sentence, including all the way to the period, is the last item in this list, correct? This requires an "and" before if this is the case. If not, I'm having trouble interpreting the true meaning here.

File: Asgardia.Constitution.Proposal.txt [hunk 4]
```diff
@@ -399,8 +398,8 @@ Article 34. Parliament
 § 10. Upon return of any vetoed law by the Head of State, Parliament may still enact that law if a super-majority of three-fourths (3/4) of Parliament vote in favour of it.
 § 11. Parliament shall be housed by no more than one member for every 5,000 Citizens and no less than one member for every 10,000 Citizens. The Parliament may not be housed with less than 50 Parliament members.
 § 12. Except in cases provided by law, members of Parliament shall be exempt from apprehension while the Parliament is in session, and any members apprehended before the opening of the session shall be freed during the term of the session upon demand of the Council of Supreme Values.
-§ 13. An ordinary session of the Parliament shall be convened twice (2) per Earth-year : one in April and the second in October.
-§ 14. The Council may determine to convene extraordinary sessions of Parliament. When a quarter (1/4) or more of the total members of Parliament makes the demand, The Council must agree to such convocation.
+§ 13. An ordinary session of the Parliament shall be convened two (2) times per Earth-year; the first session shall take place in April and the second in October.
```
Comment (BloodyClean) [inline Asgardia.Constitution.Proposal.txt:401]: Trying to match the grammar in the rest of the document and make it clearer

File: Asgardia.Constitution.Proposal.txt [hunk 5]
```diff
@@ -545,7 +542,7 @@ Article 47. Special Procedure for the Election of a New Head of State
...
-In the absence of Asgardian laws and for the purposes of implementing the provisions of the Constitution before the election of Parliament and formation of the Government of Asgardia, the Head of State may issue decrees which remain in force until appropriate laws are adopted. All decrees will be reviewed by the Council of Supreme Values and the Court of Justice. Any decree may be revoked by the Council of Supreme Values and the Court of Justice, provided that a two-thirds (2/3) qualified majority stand in agreement from both state bodies.
+In the absence of Asgardian laws and for the purposes of implementing the provisions of the Constitution before the election of Parliament and formation of the Government of Asgardia, the Head of State may issue decrees which remain in force until appropriate laws are adopted. All decrees will be reviewed by the Council of Supreme Values and the Court of Justice. Any decree may be revoked by the Council of Supreme Values and the Court of Justice, provided that a two-thirds (2/3) qualified-majority stand in agreement from both state bodies.
```
Comment (BloodyClean) [inline Asgardia.Constitution.Proposal.txt:545]: Matching the rest of the document with "qualified-majority". No qualms either way with or without the hyphen, but we should keep it the same throughout the document

File: Asgardia.Constitution.Proposal.txt (hunk 1 above)
Comment (lorezyra) [reply Asgardia.Constitution.Proposal.txt:78]: Okay. I suppose we could leave the other _discovered alien intelligence_ to be defined in substantive law?

Comment (BloodyClean) [reply Asgardia.Constitution.Proposal.txt:78]: Amusingly, this is the one I was struggling with trying to figure out how to phrase. I tried to capture this with "peoples", but I know that might not get the spirit of the message.

File: Asgardia.Constitution.Proposal.txt (hunk 3 above)
Comment (lorezyra) [reply Asgardia.Constitution.Proposal.txt:361]: should list out like this...
appoints the:

- Supreme Justice of the Judicial Court
- the Chairperson of the National Audit Office
- the Chairperson of the Council of Supreme Values
- Asgardian ambassadors extraordinary and plenipotentiary to _Earth nation states, and any other space nation states_
- 

Seems I have an extra comma here. :)

Comment (BloodyClean) [reply Asgardia.Constitution.Proposal.txt:361]: Agreed :)

**Pull request merged**